collect.py

```diff
@@ -241,19 +240,30 @@ def run_tests(firefox_driver, chrome_driver):
     chrome_driver.quit()
 
 
+NO_OF_THREADS = 5
 os.environ['PATH'] += os.pathsep + os.path.abspath('tools')
 os.environ['MOZ_HEADLESS'] = '1'
 os.environ['MOZ_HEADLESS_WIDTH'] = '412'
 os.environ['MOZ_HEADLESS_HEIGHT'] = '808'
 firefox_profile = webdriver.FirefoxProfile()
 firefox_profile.set_preference("general.useragent.override", "Mozilla/5.0 (Android 6.0.1; Mobile; rv:54.0) Gecko/54.0 Firefox/54.0")
-firefox_driver = webdriver.Firefox(firefox_profile=firefox_profile, firefox_binary=nightly_bin)
 chrome_options = webdriver.ChromeOptions()
 chrome_options.binary_location = chrome_bin
 chrome_options.add_argument("--no-sandbox")
 chrome_options.add_argument("--headless")
 chrome_options.add_argument("--window-size=412,732")
 chrome_options.add_argument("--user-agent=Mozilla/5.0 (Linux; Android 6.0.1; Nexus 5 Build/M4B30Z) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.83 Mobile Safari/537.36")
-chrome_driver = webdriver.Chrome(chrome_options=chrome_options)
 
-run_tests(firefox_driver, chrome_driver)
+
+def main(bugs):
+    firefox_driver = webdriver.Firefox(firefox_profile=firefox_profile, firefox_binary=nightly_bin)
+    chrome_driver = webdriver.Chrome(chrome_options=chrome_options)
+    run_tests(firefox_driver, chrome_driver, bugs)
+
+
+if __name__ == '__main__':
+    random.shuffle(bugs)
+    with ThreadPoolExecutor(max_workers=NO_OF_THREADS) as executor:
+        bugs_out = [bugs[init::NO_OF_THREADS] for init in range(NO_OF_THREADS)]
```

Comment:
Nit: Give this variable a more meaningful name, e.g. `bugs_sublists` or something like this.
Or you could also do this instead:
```Python
for i in range(THREADS_NUM):
    executor.submit(main, bugs[i::THREADS_NUM])
```